traefik/templates/_helpers.tpl

```diff
@@ -176,3 +176,13 @@ Key: {{ $cert.Key | b64enc }}
       {{ printf "- \"%s\"\n" . }}
     {{- end -}}
 {{- end -}}
+
+{{- define "traefik.hasPluginsVolume" -}}
+    {{- $found := false -}}
+    {{- range . -}}
+       {{- if eq .name "plugins" -}}
+           {{ $found = true }}
+       {{- end -}}
+    {{- end -}}
+    {{- $found -}}
+{{- end -}}
```

Comment:
```suggestion
```

Can this be replaced with `hasKey` :thinking: or did I miss something ?

Reply:
It's not a map but an object list.

Reply:
```
executing "traefik.podTemplate" at <.Values.deployment.additionalVolumes>: wrong type for value; expected map[string]interface {}; got []interface {}
```